api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -95,6 +95,36 @@ message RBAC {
     LOG = 2;
   }
 
+  message AuditOptions {
+    // Deny and allow here refer to RBAC decisions, not actions.
+    enum AuditCondition {
+      // Never audit.
+      NONE = 0;
+
+      // Audit when RBAC denies the request.
+      ON_DENY = 1;
+
+      // Audit when RBAC allows the request.
+      ON_ALLOW = 2;
+
+      // Audit whether RBAC allows or denies the request.
+      ON_DENY_AND_ALLOW = 3;
+    }
+
+    // Condition for the audit logging to happen.
+    // If this condition is met, all the audit loggers configured here will be invoked.
+    //
+    // [#not-implemented-hide:]
+    AuditCondition audit_condition = 1 [(validate.rules).enum = {defined_only: true}];
+
+    // Configurations for RBAC-based authorization audit loggers.
+    //
+    // [#not-implemented-hide:]
+    // [#extension-category: envoy.rbac.audit_loggers]
+    repeated core.v3.TypedExtensionConfig audit_log = 2
```

Comment:
Suggest calling this `audit_loggers`.  I think we don't need to match the semantics of the access logger anymore, since we're not configuring this in the same context.

Reply:
Makes sense. Done.